src/grammar/test/stmt-destructuring.tact

```diff
@@ -0,0 +1,18 @@
+struct S {
+    a: Int;
+    b: Int;
+    c: Int;
+}
+
+fun testFunc(): Int {
+    let s = S{ a: 1, b: 2, c: 3 };
+    let { a, b, c } = s;
+    let { a1, _, _ } = s;
+    let { _, b1, _ } = s;
+    let { _, _, c1 } = s;
+    let { a2, b2, _, } = s;
+    let { a3, _, c3, } = s;
+    let { _, b4, c4, } = s;
```

Comment:
these won't pass the scoping check, right?

Reply:
why?

Reply:
yes, I forgot to update this one after the rework

Reply:
done